tests/acceptance/bootstrap/CollaborationContext.php

```diff
@@ -355,4 +355,84 @@ public function theFollowingMimeTypesShouldExistForUser(string $shouldOrNot, Tab
 			}
 		}
 	}
+
+	/**
+	 * @Then the app list response should contain the following information:
+	 *
+	 * @param TableNode $table
+	 *
+	 * @return void
+	 * @throws Exception
+	 */
+	public function theAppListResponseShouldContainTheFollowingInformation(TableNode $table): void {
+		$responseArray = $this->featureContext->getJsonDecodedResponse($this->featureContext->getResponse());
+
+		if (!isset($responseArray['mime-types'])) {
+			Assert::fail("Expected response to contain a 'mime-types' key but contain" . print_r($responseArray, true));
+		}
+
+		$mimeTypes = $responseArray['mime-types'];
+
+		$mimeTypeMap = [];
+		foreach ($mimeTypes as $mimeType) {
+			$mimeTypeMap[$mimeType['mime_type']] = $mimeType;
+		}
+
+		foreach ($table->getColumnsHash() as $row) {
+			if (!isset($mimeTypeMap[$row['mime-type']])) {
+				Assert::fail("Expected Mime type'{$row['mime-type']}' to exist in the response" . print_r($mimeTypeMap, true) . "but doesn't exist.");
+			}
+
+			$mimeType = $mimeTypeMap[$row['mime-type']];
+			foreach ($mimeType['app_providers'] as $provider) {
+
+				$extensions = array_map('trim', explode(',', rtrim($row['target-extension'], ',')));
+				$onlyOfficeTargetExt = $extensions[0] ?? '';
+				$collaboraTargetExt = $extensions[1] ?? '';
+
+				if ($provider['name'] === 'OnlyOffice' && $onlyOfficeTargetExt) {
+					Assert::assertSame(
+						$onlyOfficeTargetExt,
+						$provider['target_ext'],
+						"Expected target_ext for OnlyOffice in mimeType '$onlyOfficeTargetExt but found '{$provider['target_ext']}"
+					);
+				}
+				if ($provider['name'] === 'Collabora' && $collaboraTargetExt) {
+					Assert::assertSame(
+						$collaboraTargetExt,
+						$provider['target_ext'],
+						"Expected target_ext for Collabora in mimeType '$collaboraTargetExt but found '{$provider['target_ext']}"
+					);
+				}
+			}
```

Comment:
we can make it short comparison like this.
```suggestion
			$targetExtensions = [];
			foreach ($mimeType['app_providers'] as $provider) {
				$targetExtensions[] = $provider['target_ext'];
			}
			$targetExtensions = \join(",", $targetExtensions);
			Assert::assertSame($row['target-extension'], $targetExtensions, "message here...")
```

Reply:
if i implement your way then it doesn't check strictly. I would like to keep it strict